data/serializer/src/main/java/tech/pegasys/teku/api/schema/merge/ExecutionPayloadCommon.java

```diff
@@ -0,0 +1,206 @@
+/*
+ * Copyright 2021 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.api.schema.merge;
+
+import static tech.pegasys.teku.api.schema.SchemaConstants.DESCRIPTION_BYTES20;
+import static tech.pegasys.teku.api.schema.SchemaConstants.DESCRIPTION_BYTES32;
+import static tech.pegasys.teku.api.schema.SchemaConstants.PATTERN_BYTES20;
+import static tech.pegasys.teku.api.schema.SchemaConstants.PATTERN_BYTES32;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.MoreObjects;
+import io.swagger.v3.oas.annotations.media.Schema;
+import java.util.Objects;
+import org.apache.tuweni.bytes.Bytes;
+import org.apache.tuweni.bytes.Bytes32;
+import org.apache.tuweni.units.bigints.UInt256;
+import tech.pegasys.teku.infrastructure.unsigned.UInt64;
+import tech.pegasys.teku.ssz.type.Bytes20;
+
+public abstract class ExecutionPayloadCommon {
+
+  @Schema(
+      name = "parent_hash",
+      type = "string",
+      format = "byte",
+      pattern = PATTERN_BYTES32,
+      description = DESCRIPTION_BYTES32)
+  public final Bytes32 parentHash;
+
```

Comment:
i think all of these have come out incorrectly because of the abstract class usage. i'm not sure how you'd get around that, but the swagger is all busted for this class...

Reply:
I guess you are not seeing the latest version, can you do a clean rebuild?